### PR TITLE
Added bias vector and regularizers

### PR DESCRIPTION
**Pull request by vsheska**
Added options to include a bias vector, and regularizers to the kernels

### CODE REVIEW

File: keras_lmu/layers.py [hunk 1]
```diff
@@ -279,6 +305,8 @@ def get_config(self):
                 input_to_hidden=self.input_to_hidden,
                 kernel_initializer=self.kernel_initializer,
                 recurrent_initializer=self.recurrent_initializer,
+                kernel_regularizer=self.kernel_regularizer,
+                recurrent_regularizer=self.recurrent_regularizer,
```
Comment (hunse) [inline keras_lmu/layers.py:309]: Need to add `use_bias` here.

File: keras_lmu/tests/test_layers.py [hunk 2]
```diff
@@ -447,3 +447,48 @@ def test_no_input_kernel_dimension_mismatch(fft):
...
+    if fft:
+        if bias:
+            assert np.isclose(diff, 0.06)
+        else:
+            assert np.isclose(diff, 0.04)
+    else:
+        if bias:
+            assert np.isclose(diff, 0.22)
+        else:
+            assert np.isclose(diff, 0.2)
```
Comment (hunse) [inline keras_lmu/tests/test_layers.py:494]: Where do these target numbers (e.g. 0.2) come from?

Comment (hunse) [reply keras_lmu/tests/test_layers.py:494]: Figured it out. They're based on the default L1 amount of regularization, and the sizes and values of the respective kernels and bias term. I've redone the test to compute their values based on all those terms.

**Comment by hunse** [comment]
I've made a few modifications. The main one is that the bias is no longer a subsidiary of the kernel, because a user may want to have a bias for the recurrent kernel when they have the encoding kernel turned off.

I also added a separate initialization and regularization term for the bias, since these are typically treated separately (e.g. in `Dense` layers), and one usually wants a different initializer for the biases (i.e. zeros) than for the weights. The `use_bias` argument is now a little redundant, in that we could accomplish this by just having the user set `bias_initializer` to `None` if they don't want a bias (this is what we do with the kernel). However, I kept it because we want the bias to be off by default, but if the user wants to turn the bias on, it's nice to have a sensible default for the initializer (i.e. zeros). Otherwise, any user that wants to turn the bias on would need to know what initializer they want to use (though we could help them out in the docstring by suggesting "zeros").

**Comment by vsheska** [comment]
Changes are looking good to me!

**Review by hunse — APPROVED** [review]

**Pull request merged**